### PR TITLE
Feature/ismooth

### PR DESCRIPTION
**Pull request by whaeck**
Following changes in NJOY2016's input file by adding the ismooth parameter to ACER and GROUPR, the input verification in NJOY21 has been updated.

Njoy ;-)

### CODE REVIEW

File: src/njoy21/input/ACER/Card6/Ismooth/test/Ismooth.test.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+SCENARIO( "Ismooth  output values",
+         "[ACER],[Card6], [Ismooth]"){
+
+  long ln{0};
+  GIVEN( "valid ismooth values" ){
+    std::vector<int> validValues{0, 1};
+
+    THEN( "the returned class has the correct value" ){
+      for( auto& ismooth : validValues ){
```
Comment (jlconlin) [inline src/njoy21/input/ACER/Card6/Ismooth/test/Ismooth.test.cpp]: @nicholas-sly has discovered that this `for` loop needs to be outside the `THEN` and the string passed to the `THEN` needs to be unique or `Catch++` will assume it's the exact same test.

File: src/njoy21/input/GROUPR/Card2/Ismooth/test/Ismooth.test.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+SCENARIO( "ismooth output values",
+         "[GROUPR],[Card2], [Ismooth]"){
+
+  long ln{0};
+  GIVEN( "valid ismooth values" ){
+    std::vector<int> validValues{0, 1};
+
+    THEN( "the returned class has the correct value" ){
+      for( auto& ismooth : validValues ){
```
Comment (jlconlin) [inline src/njoy21/input/GROUPR/Card2/Ismooth/test/Ismooth.test.cpp]: Move `for` loop outside of `THEN`.

**Comment by jlconlin** [comment]
@whaeck What is the status of this Pull Request?

**Comment by whaeck** [comment]
Nick implemented these changes independently in his pull request #66 which is why this one can be closed.

**Pull request closed without merging**